home/ivp-planning/machines/selecting-and-moving-objects.md

```diff
@@ -8,26 +8,41 @@ How to select and move objects (german)
+In the [2D-panel](../user-interface/the-2d-panel.md), you can identify selected objects by a blue tint and a white frame around the objects that displays their x- and y-size. Parts of this frame can also be used to [move](move-objects.md#moving-objects-in-the-2d-panel) and [rotate](scale-and-rotate-objects.md#rotating-objects-in-the-2d-panel) objects in 2D.
+
 
-![](../../../.gitbook/assets/iVP\_guide\_select\_objects\_2D.jpg)
+![](../../../.gitbook/assets/2d-select-highlighted.png)
 
-In the [3D-Panel](../user-interface/the-3d-panel.md), selected Objects will have blue angle brackets around them which form a virtual box. Additionally, a handle will appear in their geometric center which can also be used to [move](move-objects.md#moving-objects-in-the-3d-panel), [rotate ](scale-and-rotate-objects.md#rotating-objects-in-the-3d-panel)or [scale ](scale-objects.md#scaling-objects-in-the-3d-panel)objects in 3D.
+In the [3D-Panel](../user-interface/the-3d-panel.md), selected Objects will have blue angle brackets around them which form a virtual box. Additionally, a handle will appear in their geometric center which can also be used to [move](move-objects.md#moving-objects-in-the-3d-panel), [rotate](scale-and-rotate-objects.md#rotating-objects-in-the-3d-panel) or [scale](scale-objects.md#scaling-objects-in-the-3d-panel) objects in 3D.
 
-![](../../../.gitbook/assets/iVP\_guide\_select\_objects\_3D.jpg)
 
-## Selecting objects via the machine list panel:
+![](../../../.gitbook/assets/3d-select-brackets.png)
 
-Every object in the scene also has an entry in the [machine list panel](../user-interface/the-machine-list.md). You can select objects by **clicking on their entry**. The entries of selected machines can be identified by their blue tint.
+## Selecting objects via the Hierarchy
+
+Every object in the scene also has an entry in the [Object Hierarchy](../user-interface/the-machine-list.md). You can select objects by **clicking on their entry**. The entries of selected machines can be identified by their blue tint.
 
 {% hint style="info" %}
 It is also possible to select multiple objects by holding the **Ctrl** key while clicking on them one after another. You can also use the **Shift** key to select a range of entries at once.
 {% endhint %}
 
-![](../../../.gitbook/assets/iVP\_guide\_select\_objects\_machine\_list\_panel.jpg)
+![](../../../.gitbook/assets/2d-select-hierarchy.png)  
+
+## Box Selection
+
+It is also possible to select multiple objects in the [3D-panel](../user-interface/the-3d-panel.md) by holding down the **Ctrl*** button and dragging your mouse over the objects you want to highlight.  
+While you have multiple objects selected you can also hold the **Ctrl** button down and left click objects that you don't want to select any more.
+    
+![](../../../.gitbook/assets/3d-box-selection.png)  
+
+### Additive Selection
```

Comment:
Let's change this header to something that also reflect's the deselection you mention below.